QuizApp/Login/LoginClient.swift

```diff
@@ -4,14 +4,21 @@ protocol LoginClientProtocol {
 
     func login(password: String, username: String) async throws -> LoginResponseModel
 
-    func executeURLRequest(URLrequest: URLRequest) async throws -> LoginResponseModel
+    func checkAccessToken(accessToken: String) async throws
 
 }
 
 class LoginClient: LoginClientProtocol {
 
     let baseURL = "https://five-ios-quiz-app.herokuapp.com/"
     let loginPath = "api/v1/login"
+    let checkPath = "api/v1/check"
+
+    private let apiClient: ApiClientProtocol!
```

Comment:
It does not have to be implicit optional since it will never be nil. 

These are used in ViewControllers and Views because we don't override `init`, we just make `convenience init`.`convenience` is not required for that object to be created so we implicitly unwrap properties in case someone calls a different `init`, the app crashes since we want our init to be called.
With custom classes (that are not UIViewController and UIView), we're creating the only init and thus we don't need to implicitly unwrap properties.